features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -78,3 +78,116 @@ Feature: SDN compoment upgrade testing
       | curl | -s | --connect-timeout | 5 | <%= cb.pod2ip %>:8080 |
     Then the step should fail
     And the output should not contain "Hello"
+
+
+  # @author asood@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the namespace networkpolicy for an application works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | test1 |
```

Comment:
hmm, How about using `policy-upgrade1` and `policy-upgrade2` here in case the cluster already have `test1` and `test2` project

Reply:
Sure the name the folders can be changed.